pandas/io/pickle.py

```diff
@@ -33,7 +34,27 @@ def to_pickle(obj, path, compression='infer', protocol=pkl.HIGHEST_PROTOCOL):
         .. [1] https://docs.python.org/3/library/pickle.html
         .. versionadded:: 0.21.0
 
-
+    Examples
+    --------
+    >>> from pandas import DataFrame, read_pickle
+    >>> original_df = DataFrame({"foo": range(5), "bar": range(5, 10)})
+    >>> original_df
+       foo  bar
+    0    0    5
+    1    1    6
```

Comment:
can you add a See Also and ref read_pickle (to_hdf, to_sql, to_parquet) also good

Reply:
added 💯